### PR TITLE
Makefile: setup commands.

### PR DESCRIPTION
**Pull request by tarsisazevedo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+install:
+	pip install -r requirements.txt
+
+loaddata:
+	python manage.py loaddata deck/fixtures/user.json
```
Comment (mauricioabreu) [inline Makefile]: Try to replace these 4 lines by `python manage.py loaddata deck/fixtures`.

Comment (tarsisazevedo) [inline Makefile]: it's not working because these features need a specific order to run.

Comment (mauricioabreu) [inline Makefile]: Yep, good catch!

**Comment by lamenezes** [comment]
This looks good :+1:

File: README.md [hunk 2]
```diff
@@ -15,13 +15,9 @@ Speakerfight is an arena where speakers can fight ~against~ each other and peopl
 $ git clone git@github.com:luanfonceca/speakerfight.git
 $ cd speakerfight
 $ pip install -r requirements.txt
```
Comment (luanfonceca) [inline README.md]: What about a `make deps` to install this? Or "deps" looks like another kind of dependencies, not only python deps i mean.

Comment (vmesel) [inline README.md]: I think it should stay the way it is now! ```make deps``` will make people believe that they didn't install any other dependencies!

**Comment by tarsisazevedo** [comment]
I'll update this PR with speakerfight master, so we merge it. Ok?

**Comment by wagnerluis1982** [comment]
Hi, I'm late in this discussion, but I have a question: why `make install` only do the `pip` requirements stuff?

I think the install target should have every command described in the README with exception of `python manage.py runserver`.

**Comment by tarsisazevedo** [comment]
@wagnerluis1982 it's a good pratice in Makefiles to create target that executes only one task and then compose it on more complex tasks.
We can add another target, let's say `run` target, and it runs `setup` target and then run `python manage.py runserver`.

What do you think?

**Comment by wagnerluis1982** [comment]
@tarsisazevedo, sorry, but I didn't see that you added a `setup` target that does exactly what I intended. 

The names look strange for me. IMO the `install` target should change place with `setup`, so would be:

```
install: setup db loaddata
```

That's only my opinion, if you don't agree I'll not be angry :smile:

**Comment by luanfonceca** [comment]
can i merge this? @tarsisazevedo

**Pull request merged**